scripts/convert-typo.ts

```
@@ -0,0 +1,107 @@
+    const num = parseFloat(str);
+    return `${num / 100}em`;
+  }
+  return str;
+}
+
+function makeClassName(group: string, key: string) {
+  return `.text-${group.toLowerCase()}-${key.replace(/_/g, '-')}`;
+}
+
+// 참조 해석 유틸
+function resolveRef(obj: any, ref: string) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)